js/message_test.js

```diff
@@ -184,14 +180,7 @@ describe('Message test suite', function() {
     var response = new proto.jspb.test.DefaultValues();
 
     // Test toObject
-    var expectedObject = {
-      stringField: defaultString,
-      boolField: true,
-      intField: 11,
-      enumField: 13,
-      emptyField: '',
-      bytesField: 'bW9v'
-    };
+    var expectedObject = {};
```

Comment:
After chatting a bit internally, I think our best option for the moment would be to preserve existing semantics. So all assertions related to `toObject()` should be unchanged. We can consider changing this later.